### PR TITLE
random tiny stuff

### PR DESCRIPTION
**Pull request by hoijui**
see individual commit messages.

please say if i should change something.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +47,18 @@ and at least one yellow item (goal item).
 Token strokes will only join to other token strokes. Goal strokes will only
 join to goal strokes. Other strokes will happily join to any non-token non-goal
 strokes.
+
+
+Compilataion Instructions
```
Comment (thp) [inline README.md]: Typo: Compilation

**Comment by hoijui** [comment]
ahh good spot! :-)
done.

**Comment by hoijui** [comment]
separated out into new PRs.

**Comment by thp** [comment]
Thanks for the PRs and the cleanups!

**Comment by hoijui** [comment]
cool! thank you! :-)

**Pull request closed without merging**